index.js

```diff
@@ -71,7 +75,18 @@ function prebuildify (opts, cb) {
     if (err) return cb(err)
     loop(opts, function (err) {
       if (err) return cb(err)
-
+      if (opts.optionalPackages) {
+        var description = 'Platform specific binary for ' + packageData.name + ' on ' + opts.platform + ' OS with ' + opts.arch + ' architecture'
+        fs.writeFileSync(path.join(opts.builds, 'package.json'), JSON.stringify({
+          name: packageData.name + '-' + opts.platform + '-' + opts.arch,
+          version: packageData.version,
+          os: [opts.platform],
+          cpu: [opts.arch],
+          description
+        }, null, 2))
```

Comment:
Can we copy `license` as well? Systems that collect licenses of dependencies will like that